src/components/TextTransform.tsx

```diff
@@ -15,7 +15,8 @@ const components = {
 
 type Props = TextProps<Theme> & {
   i18nKey: string
-  values?: Record<string, unknown>
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  values?: Record<string, any>
```

Comment:
I think this could be
```ts
  values?: Record<string, never>
```

Reply:
Actually I think unknown would work here, not sure why I changed it. Might have got mixed up in my changes